### DISCUSSION TITLE
[Feature] Warning command

### DISCUSSION THREAD

**Discussion by MidNightFrance** [yewshanooi/sodium#73] [Ideas]
Warnings command to see warn of a user

**Comment by Manzanitabot123** [comment]
For this implementation, it is necessary for ava bot to have a database that stores the amount, message and date of a user's warnings, that way with a command: `/view warnings <@user>`, a moderator can view the number of warnings that a user has.
As I say again, a database update is still missing, but your idea seems very good to me and I will keep it in mind for future versions. Thank you @MidNightFrance.

**Comment by yewshanooi** [comment]
Aside from that, it will also require the bot owner to host the database on their machine locally or through the cloud too

**Comment by yewshanooi** [comment]
This feature was added quite long ago using MongoDB Atlas to store the server's logs (which include warnings of users). The staff of a particular server may view the logs using the `/logs view` subcommand). My bad, I didn't check the Issues tab to update this thread. Anyway, thanks again @MidNightFrance for your suggestion.
